### ISSUE TITLE
Error when Turing 0.8.2 installed

### ISSUE THREAD

**Issue by jonasmac16** [JuliaDiffEq/DiffEqBayes.jl#125]
Hi,
with `Turing` installed at version `0.8.2` in my Julia 1.3 env it get the following error running the example problem and it goes away once `Turing` is removed:
```
TypeError: in Type, in Targs, expected Targs<:NamedTuple, got Type{Tuple{:x}}
#turing_inference#1(::Array{InverseGamma{Float64},1}, ::DiffEqBayes.var"#8#17", ::Int64, ::NUTS{Turing.Core.ForwardDiffAD{40},(),AdvancedHMC.Adaptation.DiagEuclideanMetric}, ::Array{Symbol,1}, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::typeof(turing_inference), ::ODEProblem{Array{Float64,1},Tuple{Float64,Float64},true,Array{Float64,1},LotkaVolterraTest1{var"#40#41",Nothing,Nothing,Nothing,Nothing,Nothing,Any,Any},Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}},DiffEqBase.StandardODEProblem}, ::Tsit5, ::Array{Float64,1}, ::Array{Float64,2}, ::Array{Truncated{Normal{Float64},Continuous,Float64},1}) at turing_inference.jl:83
(::DiffEqBayes.var"#kw##turing_inference")(::NamedTuple{(:num_samples,),Tuple{Int64}}, ::typeof(turing_inference), ::ODEProblem{Array{Float64,1},Tuple{Float64,Float64},true,Array{Float64,1},LotkaVolterraTest1{var"#40#41",Nothing,Nothing,Nothing,Nothing,Nothing,Any,Any},Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}},DiffEqBase.StandardODEProblem}, ::Tsit5, ::Array{Float64,1}, ::Array{Float64,2}, ::Array{Truncated{Normal{Float64},Continuous,Float64},1}) at none:0
top-level scope at DiffEqBayes_issue.jl:26
 ```
```julia
using DiffEqBayes
using DifferentialEquations
using ParameterizedFunctions
using Distributions
using ForwardDiff

f1 = @ode_def_bare LotkaVolterraTest1 begin
 dx = a*x - x*y
 dy = -3*y + x*y
end a

p = [1.5]
u0 = [1.0,1.0]
tspan = (0.0,10.0)
prob1 = ODEProblem(f1,u0,tspan,p)

σ = 0.01                         # noise, fixed for now
t = collect(range(1,stop=10,length=10))   # observation times
sol = solve(prob1,Tsit5())

randomized = VectorOfArray([(sol(t[i]) + σ * randn(2)) for i in 1:length(t)])
data = convert(Array,randomized)

priors = [Truncated(Normal(1.5,0.1),0,2)]

bayesian_result_turing_tsit5 = turing_inference(prob1,Tsit5(),t,data,priors;num_samples=500)
```

```julia
(v1.3) pkg> status
    Status `~/.julia/environments/v1.3/Project.toml`
  [c7e460c6] ArgParse v0.6.2
  [c52e3926] Atom v0.11.3
  [a134a8b2] BlackBoxOptim v0.5.0
  [336ed68f] CSV v0.5.22
  [a93c6f00] DataFrames v0.20.0
  [191d1da1] DependentBootstrap v1.1.2
  [ebbdde9d] DiffEqBayes v2.4.0
  [1130ab10] DiffEqParamEstim v1.12.0
  [0c46a032] DifferentialEquations v6.10.1
  [31c24e10] Distributions v0.21.12
  [5789e2e9] FileIO v1.2.1
  [7073ff75] IJulia v1.20.2
  [e5e0dc1b] Juno v0.7.2
  [4345ca2d] Loess v0.5.0
  [76087f3c] NLopt v0.5.1
  [86f7a689] NamedArrays v0.9.3
  [47be7bcc] ORCA v0.3.1
  [65888b18] ParameterizedFunctions v4.2.1
  [91a5bcdd] Plots v0.28.4
  [6f49c342] RCall v0.13.4
  [295af30f] Revise v2.5.0
  [f3b207a7] StatsPlots v0.13.0
  [fce5fe82] Turing v0.8.2
  [8bb1440f] DelimitedFiles 
  [9a3f8284] Random 
```

**Comment by ChrisRackauckas** [comment]
I think the discussion in https://github.com/JuliaDiffEq/DiffEqBayes.jl/pull/123 is the same thing. @mohamed82008 @xukai92 any updates?

Also pinging @Vaibhavdixit02

**Comment by mohamed82008** [comment]
On it.

**Comment by mohamed82008** [comment]
This should be fixed on #127.
